server/models/user.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose')
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken')
+
+const userSchema = new mongoose.Schema({
```

Comment:
Should we have a type field to distinguish between entrepreneurs and investors?

Reply:
I could add that, but it was my understanding that we would be adding additional fields to the user model in the issue labeled: "PUT /users/:userId #16"

Reply:
Makes sense. I'll do that then.